### PR TITLE
Colors, colors everywhere...

### PR DESCRIPTION
**Pull request by jefjabs**
Added colored console log to analyse thread processing.

### CODE REVIEW

**Comment by mxaddict** [comment]
@jefjabs
1. Please replace spaces with TABS
2. Please re-submit with only 1 commit :wink: 

PS: Pina korek jud na comments bah, bwahaha :+1:

**Comment by jefjabs** [comment]
@mxaddict 
[OK] 1. Please replace spaces with TABS
[OK] 2. Please re-submit with only 1 commit

PS: Wish granted.. haha..

**Comment by beanmoss** [comment]
murag mga korek!! bwahahaha

File: main.cpp [hunk 1]
```diff
@@ -117,6 +117,7 @@ void get_results(std::string type) {
 	auto amounts = root->find("//table/tr[position() > 1]/td[5]/text()");
 	auto winners = root->find("//table/tr[position() > 1]/td[6]/text()");
 
+    std::cout << color << "\n" ;
```
Comment (mxaddict) [inline main.cpp]: This still has spaces :-1: Bwahahaha

File: main.cpp [hunk 2]
```diff
@@ -65,7 +65,7 @@ std::string trim(std::string string) {
  * This function actually parses the results then stores them all in a CSV
  * that you can open in a spreadsheet :)
  */
-void get_results(std::string type) {
+void get_results(std::string type,std::string color) {
```
Comment (mxaddict) [inline main.cpp]: Needs a space after the comma @beanmoss, suya ka nuh? :P Bwahahaha

**Comment by beanmoss** [comment]
Hi @mxaddict , can I contribute to this awesome project? please!

**Comment by mxaddict** [comment]
@beanmoss, go, go, go. Bwahaha

**Comment by jefjabs** [comment]
The php version. -> https://github.com/rroble/pcso

**Pull request merged**